pallets/proofs-dealer/src/weights.rs

```diff
@@ -0,0 +1,76 @@
+
+//! Autogenerated weights for `pallet_proofs_dealer`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 43.0.0
+//! DATE: 2024-10-17, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `Facundos-MacBook-Pro.local`, CPU: `<UNKNOWN>`
+//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("dev")`, DB CACHE: `1024`
+
+// Executed Command:
+// ./target/release/storage-hub-node
+// benchmark
+// pallet
+// --chain
+// dev
+// --wasm-execution=compiled
+// --pallet
+// pallet_proofs_dealer
+// --extrinsic
+// *
+// --steps
+// 50
+// --repeat
+// 20
+// --output
+// pallets/proofs-dealer/src/weights.rs
+// --template
+// ../polkadot-sdk/substrate/.maintain/frame-weight-template.hbs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
+
+/// Weight functions needed for `pallet_proofs_dealer`.
+pub trait WeightInfo {
```

Comment:
I think you wrote this by hand if I am not mistaken, if that's the case, should we instead define the trait in the lib.rs of the pallet so we don't always have to write this trait after each generation?